config.json

```diff
@@ -395,6 +395,16 @@
       "unlocked_by": null,
       "uuid": "8dfc2f0d-1141-46e9-95e2-6f35ccf6f160"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "error-handling",
+      "topics": [
+        "exception_handling"
```

Comment:
Any other suggestions for possible topics? The list of topics can be found [here](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt)

Reply:
Yes, ```optional_values```